vipps-login-api.md

```diff
@@ -699,7 +699,7 @@ _Headers_
 
 The access token is received on a successful request to the [token endpoint](#oauth-20-token).
 
-[`GET:/userinfo`](https://vippsas.github.io/vipps-login-api/#/public/userinfo)
+[`GET:/vipps-userinfo-api/userinfo`][userinfo-endpoint]
```

Comment:
@thormarius  Look at this one. It didn't match up that well. I saw that, in recurring-api and ecom-api, we say `GET:/vipps-userinfo-api/userinfo`, so I used that here. And the URL is `https://vippsas.github.io/vipps-developer-docs/api/login#tag/Userinfo-API/operation/userinfoAuthorizationCode` which does not match up that nicely with the original URL. Is this ok?

Reply:
Yes, that was the question. :-)